src/backend/mod.rs

```diff
@@ -1,11 +1,13 @@
 use event;
 use theme;
 
-// Module is not named `ncurses` to avoir naming conflict
-#[cfg(feature = "ncurses")]
-pub mod curses;
 #[cfg(feature = "termion")]
-pub mod termion;
+mod termion;
+mod curses;
+
+pub use self::curses::*;
+#[cfg(feature = "termion")]
+pub use self::termion::*;
```

Comment:
Looks like it's mostly requirements to build a dummy `Printer` for the doc tests, for instance `view/scroll.rs:139`... It should probably call `Concrete::init` instead of creating an empty struct. Doh, did I do that? :S